to-json-schema.js

```diff
@@ -70,6 +88,50 @@ const exampleAttributeMapping = (typeInput, example, jsonSchemaInput) => {
   }
 };
 
+const additionalAttributesMapping = (typeInput, avroDefinition, jsonSchemaInput) => {
+  const __ret = extractNonNullableTypeIfNeeded(typeInput, jsonSchemaInput);
+  const type = __ret.type;
+  const jsonSchema = __ret.jsonSchema;
+
+  exampleAttributeMapping(type, avroDefinition.example, jsonSchema);
+
+  function setAdditionalAttribute(name) {
+    let isValueCoherent = true;
+    if (name === 'minLength' || name === 'maxLength') {
+      isValueCoherent = avroDefinition[name] > -1;
+    } else if (name === 'multipleOf') {
+      isValueCoherent = avroDefinition[name] > 0;
+    }
+    if (avroDefinition[name] !== undefined && isValueCoherent) jsonSchema[name] = avroDefinition[name];
+  }
+
+  switch (type) {
+  case 'int':
+  case 'long':
+  case 'float':
+  case 'double':
```

Comment:
I know this fall-through is intentional, but I think it should be marked/flagged as such - maybe a comment or something as a heads-up to the next person who makes changes here.

Reply:
done